src/OpenToolkit.Mathematics/Geometry/Box2.cs

```diff
@@ -208,7 +189,7 @@ public override int GetHashCode()
         /// <inheritdoc/>
         public override string ToString()
         {
-            return string.Format("({0}{4} {1}) - ({2}{4} {3})", Left, Top, Right, Bottom, ListSeparator);
+            return string.Format("({0}{4} {1}) - ({2}{4} {3})", Min.X, Min.Y, Max.X, Max.Y, ListSeparator);
```

Comment:
I'd probably display it as position/size as that's a lot easier to reason about.

Reply:
This does go back to the, do we want min/max, throughout or do we want position/size. I think it makes more sense to have min/max, as this is how the rest of the box2 is structured by now

Reply:
Btw this is still relevant, however the structure has changed from using String.Format to using a string like this $""